consistenthash/consistenthash_test.go

```diff
@@ -34,6 +34,13 @@ func TestHashing(t *testing.T) {
 		return uint32(i)
 	})
 
+	// Get a value before any key is added to the hash
+	// Should yield an empty string
+	empty_value := hash.Get("empty key")
```

Comment:
we don't use underscores in variable names in Go.

Also, what do you mean by "empty key"? The key isn't empty. Maybe just "foo"?